labonneboite/web/user/views.py

```diff
@@ -74,11 +75,51 @@ def account_delete():
     return render_template('user/account_delete.html', **context)
 
 
+@userBlueprint.route('/account/download/csv', methods=['GET'])
+@flask_login.login_required
+def personal_data_as_csv():
+    """
+    Download as a CSV the personal data of a user.
+    """
+    attachment_name = u'mes_données_personnelles.csv'
+
+    header_row = u'prénom,nom,email'
+    values = [
+        current_user.first_name,
+        current_user.last_name,
+        current_user.email,
+    ]
+    content_row = ",".join(values)
+    csv_text = "%s\r\n%s" % (header_row, content_row)
+
+    # Return csv file
+    response = make_response(csv_text)
+    response.headers['Content-Type'] = 'application/csv'
+    response.headers['Content-Disposition'] = 'attachment; filename=%s' % attachment_name
+    return response
+
+
+@userBlueprint.route('/favorites/list/download/csv', methods=['GET'])
```

Comment:
```methods=['GET'] est inutile puisque la méthode par défaut d'une route est toujours GET. (et pareil plus haut)```  
Régis Behmo (sur une de mes PR :smile: )

Reply:
will do everywhere in the codebase